core/store/orm/config_eth_gas_price_test.go

```diff
@@ -0,0 +1,45 @@
+package orm_test
```

Comment:
What's the thinking behind putting this in its own file rather than keeping it in `config_test.go`?

Reply:
I had to move it because otherwise importing cltest created a circular dependency

Reply:
Note that `config_test.go` is `package orm` not `package orm_test` because it tests private functions.

Reply:
Yeah, I noticed that and was curious about it as well.  Sounds good :+1: